uisdk/src/adyen/res/values/styles.xml

```diff
@@ -308,7 +308,7 @@
     </style>
 
     <style name="KhSQuotesListView">
-        <item name="android:background">@drawable/uisdk_quotes_list_bg</item>
```

Comment:
@mionescu95 can we remove this old background altogether or is it still used anywhere?